ledger-core/v2/insolar/payload/payload.proto

```diff
@@ -3,14 +3,22 @@ syntax = "proto3";
 package payload;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/insolar/assured-ledger/ledger-core/v2/insproto/ins.proto";
+
+option (insproto.notation_all) = true;
+option (insproto.zeroable_all) = true;
+option (insproto.context_all) = "MessageContext";
+option (insproto.message_ctx_apply_all) = "Message";
+option (insproto.register_all) = "RegisterMessageType";
+option (insproto.projection_names) = "Head";
 
 message Meta {
-    uint32 Polymorph = 16;
+    option (insproto.id) = 1008;
 
     bytes Payload = 20;
-    bytes Sender = 21 [(gogoproto.customtype) = "github.com/insolar/assured-ledger/ledger-core/v2/reference.Global", (gogoproto.nullable) = false];
-    bytes Receiver = 22 [(gogoproto.customtype) = "github.com/insolar/assured-ledger/ledger-core/v2/reference.Global", (gogoproto.nullable) = false];
-    bytes Pulse = 23 [(gogoproto.customtype) = "github.com/insolar/assured-ledger/ledger-core/v2/pulse.Number", (gogoproto.nullable) = false];
+    bytes Sender = 21 [(gogoproto.customtype) = "github.com/insolar/assured-ledger/ledger-core/v2/reference.Global", (insproto.zeroable) = false];
+    bytes Receiver = 22 [(gogoproto.customtype) = "github.com/insolar/assured-ledger/ledger-core/v2/reference.Global", (insproto.zeroable) = false];
+    uint32 Pulse = 23 [(gogoproto.casttype) = "github.com/insolar/assured-ledger/ledger-core/v2/pulse.Number", (insproto.zeroable) = false];
     bytes ID = 24;
-    bytes OriginHash = 25 [(gogoproto.customtype) = "MessageHash", (gogoproto.nullable) = false];
+    bytes OriginHash = 25 [(gogoproto.customtype) = "MessageHash", (insproto.zeroable) = false];
```

Comment:
pls, why do you need (insproto.zeroable) = false here?

Reply:
@cyraxred I changed (gogoproto.nullable) = false to (insproto.zeroable) it's just for use insproto instead gogoproto. I have a many places with same changes.